transform-message-formats/tests/test.bal

```diff
@@ -0,0 +1,187 @@
+import ballerina/http;
+import ballerina/test;
+
+final http:Client cl = check new (string `http://localhost:${port}/healthcare`);
+
+const GRAND_OAKS_HOSPITAL = "grand oak community hospital";
+const CLEMENCY_MEDICAL_CENTER = "clemency medical center";
+const PINE_VALLEY_COMMUNITY_HOSPITAL = "pine valley community hospital";
+const DEFAULT_DOCTOR = "thomas collins";
```

Comment:
I would use a variable that is set instead. Otherwise it is like being harcoded.

Reply:
These are for test implementations and we are not going to change them.

Reply:
If everything else is hardcoded we don't need a const just for this?

Reply:
But here, we are using these conts again in the mocked client. Using them without constants and hard coding is like reusing the same value again-and-again

Reply:
Even if we use a constant I would use the name, rather than default doctor.